state-reconstruct-fetcher/src/l1_fetcher.rs

```diff
@@ -62,6 +63,10 @@ struct L1Metrics {
 
     first_l1_block: u64,
     last_l1_block: u64,
+
+    log_acquisition: PerfMetric,
+    tx_acquisition: PerfMetric,
+    parsing: PerfMetric,
```

Comment:
I think we can be a bit more descriptive here:
```suggestion
    /// The mean [`Duration`] taken to procure a log from L1.
    log_acquisition_mean: PerfMetric,
    /// The mean [`Duration`] taken to procure a transaction from L1.
    tx_acquisition_mean: PerfMetric,
    /// The mean [`Duration`] taken to parse a [`CommitBlockInfo`] from a transaction.
    block_parsing_mean: PerfMetric,
```

Reply:
IMHO that's repeating itself, because it's too specific - the averaging algorithm should be documented as part of PerfMetric. We might want to change it, too - I probably would prefer rolling average, except I didn't remember how to compute it... :-)